sms-android/snippet.html

```diff
@@ -0,0 +1,240 @@
+<div class="snippet" id="firefox-android-sms-{{ snippet_id }}">
+  <style text="text/css">
+   div.switch-{{ snippet_id }}#brandLogo {
+       background-size: {{ width|default(192, true) }}px {{ height ~ 'px' if height else 'auto' }};
+   }
+
+   .switch-{{ snippet_id }}#brandLogo {
+       height: {{ height|default(192, true) }}px;
+       width: {{ width|default(192, true) }}px;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} {
+       background-color: white;
+       padding: 5px 20px;
+       border-radius: 5px;
+       box-shadow: 0 1px 3px rgba(72, 72, 72, 0.25);
+       font-family: "Open Sans", "Helvetica Neue", "Helvetica", "Arial", sans-serif;
+       font-size: 1em;
+       color: #484848;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} a {
+       color: #0073AA;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} p {
+       font-size: 1em;
+       display: block;
+       height: auto;
+       padding: 0;
+       width: auto;
+       margin: 10px 0 10px 0;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} #right-img {
+       float: right;
+       padding-left: 6px;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} #text {
+       margin-top: 0px;
+       font-size: 1.05em;
+       line-height: 1.4;
+       margin-bottom: 20px;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} h1 {
+       font-size: 1.4em;
+       margin-bottom: 0px;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} .footer {
+       font-size: 0.78em;
+       margin: 0px;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} hr {
+       border-top: 1px dotted #c3c3c3;
+       height: 1px;
+       margin: 20px 0 20px 0;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} .button, .button:link, .button:visited {
+       display: inline-block;
+       *display: inline;
+       *zoom: 1;
+       border-radius: 0.25em;
+       box-shadow: 0px 1px 0px 0px rgba(0, 0, 0, 0.2), inset 0px -1px 0px 0px rgba(0, 0, 0, 0.3);
+       background-color: #43a6e2;
+       background-color: #277ac1;
```

Comment:
Looks like this is because the default button `background-color` property in Snadstone is followed by a vertical gradient mixin, which also adds another `background-color` occurrence as a fallback. This certainly isn't needed, and you can remove the extra property here.

https://github.com/mozilla/bedrock/blob/master/media/css/sandstone/buttons.less#L14